### DISCUSSION TITLE
Docsy theme hero image

### DISCUSSION THREAD

**Discussion by needocdude** [google/docsy#1570] [General]
Hello
I am using the Docsy theme and am adding a hero image to the index page. Where do I add the image to index.html? If you can provide an example that would be great. Below is my existing index.html code:

+++

title = “Goldydocs”

linkTitle = “Goldydocs”

+++

{{< blocks/cover title=“Welcome” image_anchor=“top” height=“full” color=“orange” >}}

<a class=“btn btn-lg btn-primary mr-3 mb-4” href=“{{< relref “/docs” >}}”>

Learn More

{{< /blocks/cover >}}

**Comment by fekete-robert** [comment]
Hi, put it into the content/ folder, and make sure to include "background" in its filename. For example, "featured-background.png"

**Comment by fekete-robert** [comment]
Sorry, correction: if you are using a language subdir under content, then put it there, like here: https://github.com/google/docsy-example/tree/main/content/en
